src/screens/Favorites/Favorites.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Text, View, ScrollView } from "react-native";
+import { useSelector } from "react-redux";
+import styles from './Favorites.style';
+import Card from "../../components/Card/Card";
+
+const Favorites = () => {
+    const favoriteList = useSelector((state) => state.Favorite);
+    
+    const FavoriteListComponent = () => (
```

Comment:
having components inside components is conventionally not recommended but it is not wrong,
Usually tho, we either move the components outside the component's code
Or we rename to "renderFavoriteListComponent" and we use it "renderFavoriteListComponent()"

But that is just conventions and has negligible impact on performance or readabiltiy

Reply:
I agree, so I moved them outside the component code.